packages/workers.new/src/index.ts

```diff
@@ -46,6 +46,7 @@ const redirects: Redirects = {
 		"Workers WebSocket Durable Objects",
 	],
 	"/worktop": ["worker-worktop", "src/index.ts", "Workers Worktop"],
+	"/kv-shortener": ["worker-kv-shortener", "src/index.ts", "Workers KV Shortener"],
```

Comment:
I think a better name for this would be `worker-url-shortener-kv` (or perhaps just `worker-url-shortener`) to make it more clear this is a URL shortener. 

This change also means we'd need to rename the directory and all the other references to the current name.